sdk/src/anima.ts

```diff
@@ -377,19 +380,57 @@ export class Anima {
       requestBody,
       handler,
       "codegen",
-      signal,
+      signal
+    );
+  }
+
+  async generateCodeFromWebsite(
+    params: GetCodeFromWebsiteParams,
+    handler: GetCodeFromWebsiteHandler = {},
+    signal?: AbortSignal
+  ) {
```

Comment:
Created a new method to maintain maximum compatibility with the existing `generateCode` call